### PR TITLE
Upgrade registration pod to use identity store

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: casepro/settings_production.py [hunk 1]
```diff
@@ -151,9 +151,17 @@ def parse_channel_info(data):
...
-                            'replace-with-auth-token'),
+    'hub_api_url': os.environ.get('REGISTRATION_URL', ''),
+    'hub_token': os.environ.get('REGISTRATION_AUTH_TOKEN',
+                                'replace-with-auth-token'),
+    'identity_store_api_url': os.environ.get(
+        'IDENTITY_STORE_API_URL',
+        'https://identity-store/api/v1/',
+    ),
+    'identity_store_token': os.environ.get(
+        'IDENTITY_STORE_TOKEN',
```
Comment (rudigiesler) [inline casepro/settings_production.py]: Maybe this should be `IDENTITY_STORE_AUTH_TOKEN`, to make it match with the existing env var for the registration service?

Comment (alexmuller) [reply casepro/settings_production.py]: Updated, thanks!

Comment (KaitCrawford) [reply casepro/settings_production.py]: Aren't the settings currently `IDENTITY_AUTH_TOKEN` and `IDENTITY_API_ROOT` ?

Comment (alexmuller) [reply casepro/settings_production.py]: Oh cool we have these settings already in the CasePro app. That makes this easier.

**Pull request merged**